src/Settings/AppearancePage.cpp

```diff
@@ -23,8 +23,9 @@
 #include <generated/SettingsInfo.hpp>
 
 AppearancePage::AppearancePage(QWidget *parent)
-    : PreferencesPageTemplate({"Locale", "Editor Theme", "Editor Font", "Test Cases Font", "Message Logger Font",
-                               "Opacity", "Show Compile And Run Only", "Display EOLN In Diff", "Extra Bottom Margin"},
+    : PreferencesPageTemplate({"Locale", "UI Style", "Editor Theme", "Editor Font", "Test Cases Font",
```

Comment:
"Application Style" is a better name.

Reply:
"UI Style" looks good when it's right under "UI Language". And I think "UI Style" is more specific. Why do you think "Application Style" is better?

Reply:
Just to be consistent with "Editor Theme", "Application Style", So names are <Scope> <property>. If you expand UI Style, (User Interface Style) is sounds too generic. How about "Application UI Style"? Or "App UI Style"?

Reply:
> Just to be consistent with "Editor Theme", "Application Style", So names are `<Scope> <property>`. If you expand UI Style, (User Interface Style) is sounds too generic. How about "Application UI Style"? Or "App UI Style"?

How can "UI" be more generic than "Application"?

Reply:
UI and Application both are generic, I have no issue with either, it's just that abbreviations in Settings is not something I prefer.As you prefer.

Reply:
> abbreviations in Settings is not something I prefer

We have tooltips for the complete name.